qemu/tests/rng_hotplug.py

```diff
@@ -76,20 +80,31 @@ def stop_rngd(vm):
                 raise exceptions.TestError(output)
             session.close()
 
+    def check_and_run_subtest(sub_test):
+        """
+        Run subtest when it's not None
+        :param sub_test: subtest name
+        """
+        if sub_test:
+            if sub_test == "rng_bat":
+                restart_rngd(vm)
```

Comment:
Will you add other sub_test? if yes, pls add a comment like "# to do: xxx", then others will not confuse this part.

Reply:
Yes,the subtest can be rng_bat,and reboot/shutdown/system_reset.
Add one line to explain about it.
Thanks